src/Akeneo/Platform/Bundle/CommunicationChannelBundle/Resources/public/components/panel/card/Link.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from 'styled-components';
+import {AkeneoThemedProps} from '@akeneo-pim-community/shared';
+
+const __ = require('oro/translator');
```

Comment:
And then if you use the `legacy-bridge` provider you could
```suggestion
import {useTranslate} from '@akeneo-pim-community/legacy-bridge';
```
And use it as `const translate = useTranslate();`